doc/01-About.md

```diff
@@ -0,0 +1,13 @@
+# Icinga Generic Trouble Ticket System
+
+Generic TTS implements Icinga Web 2's ticket hook for replacing ticket patterns
+with links to your trouble ticket system (TTS).
+Icinga Web 2's core module `monitoring` for example uses the ticket hook for
```

Comment:
Omit **core** and do not mention that other modules can use the ticket hook as well, since the module only provides an implementation and not the hook itself.